src/components/Navbar/Navbar.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import styles from './Navbar.module.css';
+import Navbar from 'react-bootstrap/Navbar';
 import web from 'assets/talawa-logo-lite-200x200.png';
```

Comment:
Change the name of the web to logo

Reply:
@yasharth291 you put this name so I decided not to change

Reply:
please change now